src/dao/endpoint.dao.ts

```diff
@@ -9,4 +9,10 @@ export class EndpointDAO extends BaseDAO {
     super(EndpointModel);
     this.model = EndpointModel;
   }
+
+  async getALLEndpointsByMicroserviceId(microserviceId: string) {
```

Comment:
as discussed the plugin will only use config (orgId,microServiceId,projectId) so we should not. let user pass it in functions.
1.create instance of  UserAccountService
2. FetchUSerAccountInfo (orgId,projectId,microServiceId)